styles.css

```diff
@@ -0,0 +1,58 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 46px 24px;
+  gap: 12px;
+  position: absolute;
+  width: 375px;
+  height: 604px;
+  top: 92px;
+  background: url(/images/header-shapes\ mobile@2x.png);
```

Comment:
[OPTIONAL]  Please use background-image property and make sure the background image is working, I would suggest that you rename your image files and replace spaces with '-' or underscore('_') etc.